### PR TITLE
[bitnami/redmine] Update MariaDB to 14.x.x

### PR DESCRIPTION
**Pull request by Mauraza**
### Description of the change

Update MariaDB

### Checklist

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [x] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [x] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

**Review by dgomezleon — CHANGES_REQUESTED** [review]
You should add a note in the Upgrade section

**Comment by Mauraza** [comment]
> You should add a note in the Upgrade section

I updated with the new version of MariaDB

File: bitnami/redmine/README.md [hunk 1]
```diff
@@ -556,6 +556,10 @@ Find more information about how to deal with common errors related to Bitnami's
 
 ## Upgrading
 
+### To 25.0.0
+
+This major release bumps the MariaDB version to 11.1. No major issues are expected during the upgrade.
```
Comment (dgomezleon) [inline bitnami/redmine/README.md:561]: Could you please use this format?

This major release bumps the MariaDB version to 11.1. Follow the [upstream instructions](https://mariadb.com/kb/en/upgrading/) for upgrading to MariaDB 11.1. No major issues are expected during the upgrade.

Comment (Mauraza) [reply bitnami/redmine/README.md:561]: But there are no existing instructions 11.0 and 11.1. In the other versions exist instructions.

Comment (dgomezleon) [reply bitnami/redmine/README.md:561]: That's why I suggest using the general link instead of the specific one.

Comment (Mauraza) [reply bitnami/redmine/README.md:561]: In other cases that is not added, why in this case is necessary?

Comment (dgomezleon) [reply bitnami/redmine/README.md:561]: MariaDB always adds a link. You can check other README. Let's handle the unification of the Upgrade Notes in a different PR.

**Pull request merged**